<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendForm</class>
 <widget class="QWidget" name="sendForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP发送测试终端</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_IP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1:9090</string>
   </property>
   <property name="placeholderText">
    <string>0.0.0.0:9090</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_message">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>接收到的消息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fileName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IP_Listen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1:9090</string>
   </property>
   <property name="placeholderText">
    <string>0.0.0.0:9090</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>UDP监听</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startListen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopListen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
